docs/js/templateData.js

```diff
@@ -17,8 +17,8 @@ limitations under the License.
 
 // Define variables for doc templates
 var context={
-    "version": "11",
-    "dotVersion": "1.1",
-    "fullDotVersion": "1.1.0",
-    "scalaVersion": "2.11"
+    "version": "22",
```

Comment:
Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)